src/output/plugins/SndioOutputPlugin.cxx

```diff
@@ -140,6 +132,9 @@ SndioOutput::Open(AudioFormat &audio_format)
 		throw std::runtime_error("Requested audio params cannot be satisfied");
 	}
 
+	sio_setvol(sio_hdl, raw_volume);
```

Comment:
Why modify the volume while opening the output? No other plugin does that.

Reply:
Because continuing playback after stop will open a new sndiod stream which will have a new 100% volume assigned to it. Could be a loud surprise.

Reply:
So there is nothing inside sndio which remembers the volume? Then this line is ok, but add a code comment explaining it.